src/Controls/samples/Controls.Sample.Profiling/Android/environment.device.txt

```diff
@@ -0,0 +1 @@
+DOTNET_DiagnosticPorts=127.0.0.1:9000,suspend
```

Comment:
Environment variables for the device. This port is the port _on_ the device and is forwarded to 9001 on the host via adb.